lib/usd/ui/layerEditor/layerTreeView.cpp

```diff
@@ -356,7 +362,7 @@ void LayerTreeView::onLockLayer(const QString& undoName) const
 
     UndoContext context(params.commandHook, params.name);
     for (auto item : *params.selection) {
```

Comment:
Any reason we go through the selection for lock layer and sub-layers, but not for lock-layer above?